addons/spreadsheet/static/src/list/plugins/list_core_plugin.js

```diff
@@ -117,8 +117,10 @@ export class ListCorePlugin extends OdooCorePlugin {
                 break;
             }
             case "DUPLICATE_ODOO_LIST": {
-                const { listId, newListId } = cmd;
-                this._addList(newListId, deepCopy(this.lists[listId].definition));
+                const { listId, newListId, duplicatedListName } = cmd;
+                const duplicatedList = deepCopy(this.lists[listId].definition);
+                duplicatedList.name = duplicatedListName ?? duplicatedList.name + " (copy)";
+                this._addList(newListId, duplicatedList);
```

Comment:
> There's no need for this if the (copy) is already added in the `DUPLICATE_ODOO_LIST` command in the Enteprise PR right ? 

Taking translated `duplicatedListName` from the command as LUL explain here [PR](https://github.com/odoo/o-spreadsheet/pull/5010)


>And I don't quite get the `duplicatedList.name = duplicatedListName ?? duplicatedList.name + " (copy)";`, if there is no name in the command (do this ever happens ?) the name becomes `undefined (copy)` ? That doesn't sound right.

Previously, it wasn't included, so I assumed it as optional (need to modify all tests using this command may be). I'm not sure if the command ever lacks a `duplicatedListName`, but if it does, the name should default to the `original list name + "(copy)"` rather than `undefined (copy)`

Reply:
Oh yeah my bad I misread, duplicatedList.name is not the same as duplicatedListName